osu.Game/Graphics/Containers/Markdown/OsuMarkdownQuoteBlock.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Markdig.Syntax;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Game.Overlays;
+
+namespace osu.Game.Graphics.Containers.Markdown
+{
+    public class OsuMarkdownQuoteBlock : MarkdownQuoteBlock
+    {
+        private Drawable background;
+
+        public OsuMarkdownQuoteBlock(QuoteBlock quoteBlock)
+            : base(quoteBlock)
+        {
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OverlayColourProvider colourProvider)
+        {
+            background.Colour = colourProvider.Content2;
+        }
+
+        protected override Drawable CreateBackground()
+        {
+            return background = base.CreateBackground();
+        }
```

Comment:
should have width 2 ([example in the wild](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns), [relevant css rule](https://github.com/ppy/osu-web/blob/376cac43a051b9c85ce95e2c446099be187b3e45/resources/assets/less/base.less#L7-L10))